### PR TITLE
Gh 1899 - Scan Connection QR code

### PR DESCRIPTION
**Pull request by biafra23**
Handles #1899

Changes proposed in this pull request:
- Move Learn more to UIBar
- Add icon to launch butto
- Add text to scanner

1st screen | scanner screen
--------- | ----------------
image | image

- **Missing**: Rounded corners

### CODE REVIEW

File: Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingHeaderView.swift [hunk 1]
```diff
@@ -8,30 +8,21 @@
...
-    }
-
     override func awakeFromNib() {
         super.awakeFromNib()
         titleLabel.setStyle(.title3)
-        scanButton.setText("Scan QR Code", .filled)
-        learnMoreButton.setTitle("", for: .normal)
-        learnMoreLabel.hyperLinkLabel(linkText: "Learn more")
-        url = App.configuration.help.desktopPairingURL
+//        scanButton.setText("Scan Code2", .filled)
```
Comment (biafra23) [inline Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingHeaderView.swift]: How do I show image + text?

Comment (biafra23) [reply Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingHeaderView.swift]: Or: How do I make the icon white?

File: Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift [hunk 2]
```diff
@@ -64,12 +79,14 @@ class DesktopPairingViewController: UITableViewController {
     }
 
     private func scan() {
-        let vc = QRCodeScannerViewController()
+        let vc = QRCodeScannerViewController(label: "Go to Gnosis Safe Web and select Connect wallet.")
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift]: passing a String doesn't add the "bold" style to the "Gnosis Safe Web" and "Connect wallet" words. The solution would be to create and pass NSAttributedString (you can create its subclass, NSMutableAttributedString). Then, you assign that string to the label by `uiLabel.attributedText = attributedLabel`

Comment (DmitryBespalov) [reply Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift]: we have some example code in the code base

**Comment by DmitryBespalov** [comment]
Let's not do rounded corners - seems to be a waste of time at this moment

File: Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift [hunk 3]
```diff
@@ -31,6 +35,12 @@ class DesktopPairingViewController: UITableViewController {
         tableView.registerHeaderFooterView(DesktopPairingHeaderView.self)
         tableView.sectionHeaderHeight = UITableView.automaticDimension
 
+        infoButton = UIBarButtonItem(image: UIImage.init(named: "ico-info"),
```
Comment (biafra23) [inline Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift]: This button looks too small. Is there an easy way to resize it?

**Review by DmitryBespalov — APPROVED** [review]
Looks good!

File: Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift (hunk 3 above)
Comment (elgatovital) [reply Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift]: isn't this part of https://app.zenhub.com/workspaces/mobile-engineering-60cc78740f488d000e996335/issues/gnosis/safe-ios/1900? The whole screen & ui adjustments

Comment (biafra23) [reply Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift]: Ups. I hope I didn't do duplicate work.

Comment (biafra23) [reply Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift]: @elgatovital I hope you can reuse my work. I thought 1900 is only about the items in the list.

**Pull request merged**